src/coreclr/ilasm/GrammarExtractor/README.md

```diff
@@ -0,0 +1,10 @@
+# Grammar extractor for IL tools
+
+Tool to extract IL grammar in `Backus-Naur Form (BNF)` from `Yet Another Compiler-Compiler (Yacc)`.
+
+Usage:
+
+```sh
+cd runtime
+./dotnet.sh run --project src/coreclr/ilasm/GrammarExtractor src/coreclr/ilasm/asmparse.y > src/coreclr/ilasm/prebuilt/asmparse.grammar
```

Comment:
It would be nice to also mention this in https://github.com/dotnet/runtime/blob/main/src/coreclr/ilasm/README.md . We want to regenerate the grammar every time the parser gets updated.